.github/workflows/propagate_to_manifestdb.yml

```diff
@@ -0,0 +1,77 @@
+name: Propagate update to OSBuild
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  propagate:
+    runs-on: ubuntu-latest
+    env:
+      SCHUTZBOT_GH: ${{ secrets.SCHUTZBOT_GH}}
+    steps:
+      - name: Checkout OSBuild
+        uses: actions/checkout@v3
+        with:
+          path: osbuild
+          fetch-depth: 0
+
+      - name: Update Manifest-db
+        run: |
+          # update only if manifest-db SHA was updated in the last commit
+          cd osbuild
+          ACTUAL_MDB_SHA=$(git show HEAD:Schutzfile | jq -r '.global.dependencies."manifest-db".commit')
+          PREVIOUS_MDB_SHA=$(git show HEAD^:Schutzfile | jq -r '.global.dependencies."manifest-db".commit')
+          if [[ $ACTUAL_MDB_SHA != $PREVIOUS_MDB_SHA ]] ; then
```

Comment:
Inverting the condition, adding `exit 0` after `echo "The manifest-db SHA was not updated in the last commit, ignoring the dependency update"` and removing `else` would remove one level of indentation.

Reply:
Good idea! Thanks.